### DISCUSSION TITLE
Archive.is blocking archiving from it :(

### DISCUSSION THREAD

**Discussion by gerroon** [ArchiveBox/ArchiveBox#1301] [Q&A] [answered]
I wanted to archive this locally , since the original is a bloomberg article.

https://archive.is/2023.12.19-110503/https://www.bloomberg.com/features/2023-california-spring-water-bluetriton/

I got this :( Is there a way to archive from that site, so I have local copies? I can access the site no problem with the same IP, so this is definitely happening because of the archiving effort.

![image](https://github.com/ArchiveBox/ArchiveBox/assets/8519469/a0f81a3e-83c7-4f9b-8ff8-8b5fa9d75e7f)

**Comment by pirate** [comment, marked as answer]
Is your DNS set to 1.1.1.1? The owner of Archive.is hates Cloudflare and famously shows this "unsolvable captcha" page to troll people using Cloudflare DNS.

Check if there's any chance your container, Docker VM, or host machine are set up to resolve DNS through 1.1.1.1.

#### The simple way

```bash
docker compose run archivebox add 'https://one.one.one.one/help/'

# then view the snapshot output to see if it shows you resolving DNS through cloudflare
```

#### The hard way

```bash
docker compose up -d
docker compose exec archivebox /bin/bash

apt update -qq
apt install -y dnsutils

nslookup archive.is
dig +trace archive.is
```

Check for any cloudflare domains or IPs in the DNS trace.

---

#### The Fix

Change the `dns:` section in `docker-compose.yml` to `8.8.8.8` to use Google's DNS servers. And/or change the `CURL_USER_AGENT`/`WGET_USER_AGENT`/`CHROME_USER_AGENT` archivebox config options to match your normal daily browser's user string (i.e. remove the ArchiveBox part from the default).

https://github.com/ArchiveBox/ArchiveBox/wiki/Configuration#curl_user_agent

```bash
# see the default user agent
archivebox config --get CHROME_USER_AGENT

# change the user agent
archivebox config --set CHROME_USER_AGENT='Mozilla/5.0 ...'
```

**Reply by gerroon** [reply]
thanks the user agent seemd to help.

**Reply by TomLucidor** [reply]
What are some of the DNS resolver that people should use then other than the Google default or CloudFlare?
